packages/mongo-nav/src/MongoNav.tsx

```diff
@@ -100,6 +100,21 @@ interface MongoNavInterface {
    * Callback that receives the response of the fetched data, having been converted from JSON into an object.
    */
   onSuccess?: (response: DataInterface) => void;
+
+  /**
+   * Whether or not a user is using Ops Manager
+   */
+  isOnPrem?: boolean;
+
+  /**
+   * Callback executed when user logs out
+   */
+  onLogout?: React.MouseEventHandler;
+
+  /**
+   * Version of Ops Manager that a user is using
+   */
+  onPremVersion?: string;
```

Comment:
(ditto reminder for an `multifactorAuthEnabled` prop)

Reply:
👍